hal/src/common/thumbv6m/sercom/v2/pad_map.rs

```diff
@@ -0,0 +1,300 @@
+//! Implementations of the [`pads`] [`Map`] trait
+//!
+//! This module provides implementations of [`Map`] for the SAMD11 and SAMD21
+//! series chips. Unlike the SAMD5x & SAME5x chips, the SAMD11 and SAMD21 chips
+//! do not have any concept of IOSET. A given SERCOM can use any combination of
+//! GPIO pins, so long as those pins can be configured as pads for that SERCOM.
+//!
+//! This module implements [`Map`] directly on [`PinId`]s. For a given
+//! [`Sercom`] and [`PadNum`], the [`Map`] trait identifies the correct
+//! [`PinMode`] for the [`PinId`].
+//!
+//! [`pads`]: crate::sercom::v2::pads
+//! [`Sercom`]: crate::sercom::v2::pads::Sercom
+//! [`PadNum`]: crate::sercom::v2::pads::PadNum
+//! [`Map`]: crate::sercom::v2::pads::Map
+//! [`PinId`]: crate::gpio::v2::PinId
+//! [`PinMode`]: crate::gpio::v2::PinMode
+use crate::gpio::v2::*;
+use crate::sercom::v2::pads::*;
+
+//==============================================================================
+//  Pad definitions
+//==============================================================================
+
+// TODO
```

Comment:
Is there something left to do here?

Reply:
I always forget to check for TODOs.

I probably wanted to add some comments for the macro. I can do that